### ISSUE TITLE
False positive

### ISSUE THREAD

**Issue by thiaguerd** [sds/slim-lint#118]
```slim
.something
  - while a >= b
    = render 'something'
```

RuboCop: Style/WhileUntilDo: Do not use `do` with multi-line `while`

version 0.20.1

**Comment by sds** [comment]
Thanks for reporting this issue, @thiaguerd.

Looks like the erroneous `do` in this situation is inserted by the Slim gem itself:

https://github.com/slim-template/slim/blob/1f3bcf011877aa93758b344628895a7716b76b6e/lib/slim/do_inserter.rb#L10-L18

I opted to add this to the list of ignored cops here: 9d4dbebda6aff052b3c33b3e8575fb6f80067af0
